schemas/stsci.edu/asdf/0.1.0/transform/regions_selector.yaml

```diff
@@ -0,0 +1,55 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/transform/regions_selector"
+tag: "tag:stsci.edu:asdf/0.1.0/transform/regions_selector"
+title: >
+  Represents a RegionsSelector transform.
+description: |
+  A model which maps regions to their corresponding transforms.
+
+allOf:
+  - $ref: transform
+  - type: object
+    properties:
+      mask:
+        description: |
+          An array with the shape of the detector/observation.
+          Pixel values are of type int or str and represent
+          region labels as in the RegionSelector transform.
+          Pixels which are not within any region have value 0 or " ".
+
+        $ref: ./selector_mask
```

Comment:
@mdboom Back to this issue. Are you questioning the names or the models? I suspect it's the names and yes, this looks a bit repetitive :). The models for these schemas are here:

https://github.com/spacetelescope/gwcs/blob/master/gwcs/selector.py

A use case for `SelectorMask` is one in which it is initialized by vertices of polygons on the detector. The inverse is a list of spherical polygons on the sky. However I'm just now starting to play with the sphere package, I can't include an example yet. So I'm proposing the following changes:

Change the name of the parameter from `RegionsSelector.mask` to `RegionsSelector.selector_mask`.  So your example above would look like this:

```
transform: !regions_selector
   selector_mask:
      mask: [...array...]
      inverse:
          selector_mask:
              mask: [...array...]
```
 In addition I think it'd be best if `SelectorMask` keeps the `vertices` parameter for easier computation of spherical polygons, so the new signature may be
 `SelectorMask(mask=None, vertices=None, geometry=None)`. I have one more use case which I meed to support and it is computing the mask using {label: vertices} and {label:transform} . So obviously the interface will change in the future and so will the schema. 
But a possible schema may look like:
```
transform: !regions_selector
   selector_mask:
      vertices: [{label: vertices_array]
      geometry: cartesian
      inverse:
          selector_mask:
              vertices: [{label: spherical_polygon_vertices}]
              geometry: spherical
```

Or are you questioning the models in which case ignore this.
Obviously this schema is driven by the implementation and use cases and those are not quite set yet.